src/activities/multiplication-tables/MultiplicationTables.qml

```diff
@@ -0,0 +1,254 @@
+            property int rectWidth: main.height/15 //main.width/20
+            property bool answer: true
+        }
+
+        onStart: { Activity.start(items) }
+        onStop: { Activity.stop() }
+
+        /////////////////////////////////////////////////////
+        
+        Grid {
+            id:gridRow
+            anchors {
+                top: parent.top
+                topMargin: 20
+                left: parent.left
+                leftMargin: 20
+            }
+            spacing: items.spacing
+            columns: 11
+            rows: 1
+            Repeater {
+                id: repeaterGridRow
+                model: 11
+                Rectangle {
+                   id: dotsGridRow
+                   color: "white"
+                   width: items.rectWidth
+                   height: items.rectHeight
+                   GCText{ text: index}
+                }
+            }
+
+        }
+
+        Grid {
+            id:gridCol
+            anchors {
+                top: parent.top
+                topMargin: 20
+                left: parent.left
+                leftMargin: 20
+            }
+            spacing: items.spacing
+            columns: 1
+            rows: 11
+            Repeater {
+                id: repeaterGridCol
+                model: 11
+                Rectangle {
+                   id: dotsGridCol
+                   color: "white"
+                   width: items.rectWidth
+                   height: items.rectHeight
+                   GCText{ text: index}
+                }
+            }
+
+        }
+
+        Grid {
+            id: grid
+            anchors {
+                top: gridRow.bottom
+                topMargin: items.spacing
+                left: gridCol.right
+                leftMargin: items.spacing
+            }
+
+            spacing: items.spacing
+            columns: 10
+            rows: 10
+
+            Repeater {
+                id: repeater
+                model: 100
+                Rectangle {
+                    id: dots
+                    width: items.rectWidth
+                    height: items.rectHeight
+                    state: "default"
+                    property bool clickedFlag: false
+                    states:[
+                        State {
+                            name: "default"
+                            PropertyChanges { target: dots; color: "green"}
+                            PropertyChanges { target: dots; clickedFlag: false}
+                        },
+                        State {
+                            name: "active"
+                            PropertyChanges { target: dots; color: "red"}
+                            PropertyChanges { target: dots; clickedFlag: true}
+                        }
+                    ]
+
+                    MouseArea {
+                        anchors.fill: parent
+                        onClicked: {
+                            if (dots.state == "default")
```

Comment:
From what I understood:
According to Montessori, aim for this activity is to count the number of squares(beads in Montessori's site) in each case, and table is explicitly given to the child for help.
Now, as he counts the number of squares by physically clicking them, he can telly his answer with the table given on the right(Point number 8 on Montessori's site), thereby relating the concept of tables with the multiplication and addition.

On the boring part, even I felt it monotonous to just punch and count squares from level 1-10, but for a specific table I think it's a good way to visualize how addition of a specific number multiple times is related to the table of that number.
**Solution:** Instead of relating tables with levels, we let the child decide which table he/she wants, just like sliding the card(Point number 3 on Montessori's site).

On making it more interactive, in which child enters the solution.
**Solution:** If tables are not related to levels, as suggested in above solution, then I can try making two levels, one in which tables are shown(just like the present game) and second in which along with counting squares by clicking them, a child needs to input solution also, eg:- instead of "4 x 5 = 20", "4 x 5 = " is given.

Please let me know your thoughts on the this.